core/kazoo_services/src/kz_services.erl

```diff
@@ -739,18 +747,18 @@ has_good_balance(Services, #{amount := Amount}=Options) ->
 has_good_balance(Balance, Amount, 'false', _) when (Balance - Amount) > 0 ->
     {'true', <<"has positive balance">>};
 has_good_balance(Balance, Amount, 'false', _) when (Balance - Amount) =< 0 ->
-    Msg = io_lib:format("has negative balance, curr_balance: ~b amount: ~b proposed_balance: ~b"
-                       ,[Balance, Amount, Balance - Amount]
+    Msg = io_lib:format("has negative balance, current balance: ~b proposed balance: ~b"
```

Comment:
@icehess maybe there is a cleaner way to convey this?

Msg = io_lib:format("debit of ~b from ~b results in a negative balance", [Amount, Balance]),